labcontrol/gui/test/test_pool.py

```diff
@@ -13,20 +13,65 @@
 
 
 class TestPoolHandlers(TestHandlerBase):
-    def test_get_plate_list_handler(self):
-        response = self.get('/pool_list')
+    def test_get_pool_list_handler_all(self):
+        response = self.get('/pool_list/all/')
```

Comment:
The '/all/' component of the resource string seems redundant - '/pool_list/' by itself, and with no further qualifying resource should display all resources by default, IMHO.

Reply:
On the other hand, the internal behavior of the handler would be implicit. I'd be more in favor of explicit behavior

Reply:
@charles-cowart Given that @wasade likes the explicit behavior--and given that I actually don't know how to tweak the tornado handler regex to do both the explicit behavior and the implicit behavior in the same handler set-up :( -- would you be willing to table this idea?

Reply:
definitely - not a problem.